starknet-curve/src/ec_point.rs

```diff
@@ -0,0 +1,296 @@
+use starknet_ff::FieldElement;
```

Comment:
We don't need all the elements of this file anymore, see https://github.com/xJonathanLEI/starknet-rs/pull/562